### PR TITLE
feat(souce-holiday-api): New connector!

### PR DESCRIPTION
**Pull request by natikgadzhi**
## What

This is a manifest-only source connector for Holiday API. I remember @btkcodedev also made one, and I taped how I made one in Builder. 

This PR is he output of `yoink shiny ~/Downloads/holiday-api.yaml` on my machine ;-) See #42132.

## TODO

This is still a draft, I want to be able to make this _fully_ automatic. To get there, I need to add `description` to the connector manifest, so we can put that blob in the docs. That's a low-code feature. I think I can add it easily, but I just haven't gotten to it yet.

### CODE REVIEW

**Comment by natikgadzhi** [comment]
Well at least docs are insta-broken.

File: airbyte-integrations/connectors/source-holiday-api/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    pypi:
+      enabled: false
+  registries:
+    cloud:
+      enabled: true
+    oss:
+      enabled: true
+  releaseStage: alpha
+  supportLevel: community
+  tags:
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-holiday-api/metadata.yaml:31]:
```suggestion
  tags:
  - language:manifest-only
```

File: airbyte-integrations/connectors/source-holiday-api/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+data:
+  ab_internal:
+    ql: 100
+    sl: 100
+  allowedHosts:
+    hosts:
+    - holidayapi.com
+  connectorBuildOptions:
+    baseImage: airbyte/source-declarative-manifest
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-holiday-api/metadata.yaml:9]: This is a bug btw, it should include the sha too.

**Comment by btkcodedev** [comment]
This latest base image SHA should work for this right?
  connectorBuildOptions:
    baseImage: docker.io/airbyte/source-declarative-manifest:3.10.2@sha256:ea85f0270dfda900a16f3dffb247002a2585a4283816edad66b89e6963d376a4
 
 I've tried it with [Source-XKCD](https://github.com/airbytehq/airbyte/pull/42479) and the build worked!!

**Comment by btkcodedev** [comment]
@natikgadzhi, I've merged the master, fixed the sha version, and formatted, Hope this works  :crossed_fingers:

**Comment by btkcodedev** [comment]
Added Icon, Fixed Readme, Added new streams from [previous PR](https://github.com/airbytehq/airbyte/pull/36986)

**Comment by btkcodedev** [comment]
Yep! The build passed as expected, No tests are configured for manifest-only connectors I think, 
But it works in normal build as checked in https://github.com/airbytehq/airbyte/pull/36986
And all checks passed!

image
image

After a check, it's ready to merge I guess 
:smile:

**Pull request closed without merging**